examples/proj_create_example.c

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+    Simple example code demonstrating use of the proj.h API for 2D coordinate
+    transformations.
+
+    It illustrates the proj_create() function. It shows how, given a
+    projection, one can convert from geodetic to projected coordinates
+    and vice-versa.
+
+    Note: This file is in-lined in the documentation. Any changes must be
+    reflected in docs/source/development/quickstart.rst
+
+*******************************************************************************/
+#include <proj.h>
+#include <stdio.h>
+#include <math.h>
+
+int main (void) {
+  PJ_CONTEXT *C;
+  PJ *P;
+
+  /* Create the context. */
+  /* You may set C=PJ_DEFAULT_CTX if you are sure you will     */
+  /* use PJ objects from only one thread                       */
+  C = proj_context_create();
+
+  /* Create the projection. */
+  P = proj_create(C, "+proj=utm +zone=32 +datum=WGS84");
+
+  if (0 == P) {
+    fprintf(stderr, "Failed to create transformation object.\n");
+    return 1;
+  }
+
+  /* Longitude and latitude of Copenhagen, in degrees. */
+  double lon = 12.0, lat = 55.0;
```

Comment:
Something along the lines of 
```suggestion
  double x = 6080642.11, y = 1886936.97;
```